target/linux/realtek/files-5.15/drivers/net/dsa/rtl83xx/qos.c

```diff
@@ -539,15 +544,17 @@ void rtl839x_config_qos(void)
 	// Re-mark DEI: 4 bit-fields of 2 bits each, field 0 is bits 0-1, ...
 	sw_w32((0x1 << 2) | (0x1 << 4), RTL839X_RMK_DEI_CTRL);
 
-	/* Set Congestion avoidance drop probability to 0 for drop precedences 0-2 (bits 24-31)
+	/*
```

Comment:
comments on netdev related stuff asks for comments starting from /* so this comment format is ok.

Reply:
Wait, you are saying that we have linus's coding style, and netdev coding style? I'm pretty sure Linus has in the past raged at people for using the wrong stye (I'd find Linus's mail if you really want it :p).

Reply:
Yep kernel checkpatch script apply different rules for comments related to netdev code. Mainly it's in the first line with multi line comments. 

Happy to read the email to have some fun with Linus getting angry ahahahah

Reply:
That is so ridiculous. I actually found the same statement in the coding style guide. Like 'wtf'. Why can you not just be consistent.

As for finding his post, needle, haystack :( It was something along the lines of 'This is how you write a comment'. Anyway, I'll remove all the 'multi-line comments starting with a near blank line' altogether from 'files/net' which is nearly everything then.